packages/flutter_tools/lib/runner.dart

```diff
@@ -25,7 +25,7 @@ import 'src/runner/flutter_command_runner.dart';
 /// Runs the Flutter tool with support for the specified list of [commands].
 Future<int> run(
   List<String> args,
-  List<FlutterCommand> commands, {
+  dynamic commands, {
```

Comment:
boom, non-breaking!

Reply:
Can you add a documentation comment for what types are expected?

Reply:
Done, with todo to fix